pwa-devdocs/src/venia-pwa-concept/setup/index.md

```diff
@@ -89,6 +89,11 @@ The Venia storefront has been verified to be compatible with the following local
 
 Don't forget to install the [Venia sample data][]!
 
+{: .bs-callout .bs-callout-info}
+In case you connected to a development Magento environment that has a self-signed SSL-certificate, you might run into an `request to [BACKEND_URL] failed, reason: self signed certificate` error when running `yarn run build` or `yarn run watch`.
+This can be circumvented by adding `NODE_TLS_REJECT_UNAUTHORIZED=0` to `packages/venia-concept/.env`.
+Or, alternatively, you could add the variable to the `yarn` command, e.g. `NODE_TLS_REJECT_UNAUTHORIZED=0 yarn build` or `NODE_TLS_REJECT_UNAUTHORIZED=0 yarn build`
```

Comment:
what is the difference between these two commands?

Reply:
Sorry for that, I updated the examples.